README.md

```diff
@@ -28,19 +28,19 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
 
 ## ➤ Table of Contents
```

Comment:
you can remove the `➤` from here as well for uniformity

Reply:
> you can remove the `➤` from here as well for uniformity

Ok, so should I remove this `➤` or retain it? The broken link problem is actually occurring due to this character.